src/en/clouds-lxd-advanced.md

```diff
@@ -214,6 +215,59 @@ Now that the cloud and credentials have been added the next step is to create a
 controller. See [Creating a controller][clouds-lxd-creating-a-controller] on
 the main LXD page.
 
+## Charms and LXD profiles
+
+Juju (`v.2.5.0`) has the following functionality for those charms that make use
+of LXD profiles:
+
+ - A validity check is performed on the profile(s) during the deployment of the
+   charm. This is based on a hardcoded list of allowed items, everything else
+   being denied. The `--force` option can be used to bypass this check but this
+   is not recommended. The list is:
+   
+```no-highlight
+config
+   -boot
+   -limits
+   -migration
+
+devices
+   unix-char
+   unix-block
+   gpu
+   usb
+```
+
+ - A profile is upgraded during the upgrade of the charm
+   (`juju upgrade-charm`).
+ - A profile is exposed at the machine level by using either the `show-machine`
```

Comment:
"Profiles are displayed at the..." ?

Reply:
the alternative is using `juju status --format=yaml`

Reply:
Thanks.